### PR TITLE
esn#12 Bootstrap the login application

### PR DESCRIPTION
**Pull request by chamerling**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
 # esn-frontend-login
 
-The login page for OpenPaaS ESN
+Login SPA for OpenPaaS.
+The application is used to allow developers to authenticate with basic auth while they are working on other 'business' SPA:
```
Comment (tuanlc) [inline README.md]:
```suggestion
The application is used to allow developers to authenticate with basic auth while they are working on other 'business' SPAs:
```

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,11678 @@
+{
```
Comment (tuanlc) [inline package-lock.json]: Git should ignore this file, no?

**Review by tuanlc — COMMENTED** [review]
You are missing the License file and license info in the package.json file.

**Review by itswillta — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chamerling) [reply README.md]: Fixed

File: package-lock.json (hunk 2 above)
Comment (chamerling) [reply package-lock.json]: Fixed

**Review by itswillta — APPROVED** [review]

**Review by tuanlc — APPROVED** [review]

**Pull request merged**